lib/routes/account.js

```diff
@@ -960,7 +970,21 @@ module.exports = function (
         log.begin('Account.deviceDestroy', request)
         var sessionToken = request.auth.credentials
         var uid = sessionToken.uid
-        db.deleteDevice(uid, request.payload.id).then(reply, reply)
+        db.deleteDevice(uid, request.payload.id)
+          .then(reply, reply)
```

Comment:
Note that I send this response before emitting the activity event, in order to work round what seems like the same problem as @seanmonstar mentioned in https://github.com/mozilla/fxa-auth-server/pull/1263#issuecomment-219892441.